config/grafana/dashboards/integration-service-health.json

```diff
@@ -0,0 +1,359 @@
+              {
+                "color": "green",
+                "value": null
+              },
+              {
+                "color": "red",
+                "value": 5
+              }
+            ]
+          }
+        },
+        "overrides": []
+      },
+      "gridPos": {
+        "h": 9,
+        "w": 24,
+        "x": 0,
+        "y": 8
+      },
+      "id": 10,
+      "interval": "60s",
+      "options": {
+        "legend": {
+          "calcs": [],
+          "displayMode": "list",
+          "placement": "bottom",
+          "showLegend": true
+        },
+        "tooltip": {
+          "mode": "single",
+          "sort": "none"
+        }
+      },
+      "targets": [
+        {
+          "datasource": "$datasource",
+          "editorMode": "builder",
+          "expr": "controller_runtime_reconcile_errors_total{namespace=\"integration-service\", controller=\"snapshot\"}",
+          "interval": "",
+          "legendFormat": "{{controller}}",
+          "range": true,
+          "refId": "A"
+        },
+        {
+          "datasource": "$datasource",
+          "editorMode": "builder",
+          "expr": "controller_runtime_reconcile_errors_total{namespace=\"integration-service\", controller=\"pipelinerun\"}",
+          "interval": "",
+          "legendFormat": "{{controller}}",
+          "range": true,
+          "refId": "B"
+        },
+        {
+          "datasource": "$datasource",
+          "editorMode": "builder",
+          "expr": "controller_runtime_reconcile_errors_total{namespace=\"integration-service\", controller=\"component\"}",
+          "interval": "",
+          "legendFormat": "{{controller}}",
+          "range": true,
+          "refId": "C"
+        },
+        {
+          "datasource": "$datasource",
+          "editorMode": "builder",
+          "expr": "controller_runtime_reconcile_errors_total{namespace=\"integration-service\", controller=\"integrationtestscenario\"}",
+          "interval": "",
+          "legendFormat": "{{controller}}",
+          "range": true,
+          "refId": "E"
+        }
+      ],
+      "title": "The reconcilation errors per controller",
+      "type": "timeseries"
+    }
+  ],
+  "refresh": "",
+  "schemaVersion": 38,
+  "style": "dark",
+  "tags": [],
+  "templating": {
+    "list": [
+      {
+        "current": {
+          "selected": false,
+          "text": "prometheus-appstudio-ds",
+          "value": "prometheus-appstudio-ds"
+        },
+        "hide": 0,
+        "includeAll": false,
+        "multi": false,
+        "name": "datasource",
+        "options": [],
+        "query": "prometheus",
+        "refresh": 1,
+        "regex": ".*-(appstudio)-.*",
+        "skipUrlSync": false,
+        "type": "datasource"
+      }
+    ]
+  },
+  "time": {
+    "from": "now-6h",
+    "to": "now"
+  },
+  "timepicker": {},
+  "timezone": "",
+  "title": "Integration Service Health",
+  "uid": "b18e4904-b2c7-4942-8df6-60ffeeb04dde",
```

Comment:
It should be updated to the `uid` on Konflux dashboard after requesting it, I think.

Reply:
Requesting what? You should be able to create dashboard on stage instances.

https://github.com/redhat-appstudio/infra-deployments/tree/main/components/monitoring

Reply:
Thanks, let me read it.

Reply:
@MartinBasti Read the [doc](https://github.com/redhat-appstudio/infra-deployments/tree/main/components/monitoring#create-a-dashboard) above, it looks we can't create dashboard for stg/prod on UI, how do you know the `id ` and `uid` when you initialize this json file? https://github.com/redhat-appstudio/integration-service/commit/ff30f45f130f51d7627c2f676449316ed25592d2#diff-feda9151b85b04e15ec8512d539944027a2dbaf5f2e7451d3c4b7145b47747a5R23

Let me leave one question in o11y team channel.

Reply:
Answers from o11y team is [here](https://redhat-internal.slack.com/archives/C04FDFTF8EB/p1712841757612479?thread_ts=1712815327.518889&cid=C04FDFTF8EB)

Reply:
thanks for checking